Sources/Versioning/Commit.swift

```diff
@@ -25,6 +25,9 @@ public struct Commit {
             self.type = CommitType(rawValue: String(trimmedExclamation))!
         } else if let type = CommitType(rawValue: String(components[0])) {
             self.type = type
+        } else if let trimmedType = components[0].components(separatedBy: "(").first,
```

Comment:
I wonder if we should use regex for this, as this logic would currently allow an invalid commit: 
`build(deps: something something` with no closing parenthesis.

Reply:
It's not a should, it's a must - will add those tests in and give it a go